src/main/java/Calculator.java

```diff
@@ -0,0 +1,83 @@
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Calculator {
+    double sum;
+    Scanner scaner = new Scanner(System.in);
```

Comment:
🍏 Поля, которые используются только внутри класса, лучше делать приватными, чтобы их нельзя было изменить снаружи:
```suggestion
    private double sum;
    private Scanner scaner = new Scanner(System.in);
```